integration/gethnetwork/geth_network.go

```diff
@@ -46,10 +44,14 @@ const (
 	websocketFlag      = "--ws" // Enables websocket connections to the node.
 	wsPortFlag         = "--ws.port"
 
+	// syncModeFlag defines the node block sync approach
+	// snap (the default) mode does not work well for small, rapidly deployed private networks
+	syncModeFlag = "--syncmode=full"
+
 	// We pre-allocate a wallet matching the private key used in the tests, plus an account per clique member.
 	genesisJSONTemplate = `{
 	  "config": {
-		"chainId": 777,
+		"chainId": 1337,
```

Comment:
@otherview 777 is strictly a better chain ID than 1337.

<img width="628" alt="Screenshot 2022-05-04 at 16 50 01" src="https://user-images.githubusercontent.com/24230167/166720443-492d69eb-f0a7-4c9a-804b-f2aadfeccb66.png">